pages/linux/nautilus.md

```diff
@@ -0,0 +1,13 @@
+# nautilus
+
+> Default file explorer for Gnome desktop environment.
+> Also known as Gnome Files.
+> More information: <https://github.com/GNOME/nautilus>.
+
+- Launch Nautilus as current user:
```

Comment:
```suggestion
- Launch Nautilus:
```
I'd say that it's obvious that nautilus is launched as the current user.

Reply:
makes sense to me. i added it to mirror the second example, but it should be pretty obvious for this example.